### PR TITLE
[MOSIP-29167]reusable workflow

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

File: .github/workflows/push-trigger.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: Maven Package upon a push
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    types: [opened]
+    branches:
```
Comment (syedsalman3753) [inline .github/workflows/push-trigger.yml:8]: Remove branches section from here

File: .github/workflows/release-changes.yml [hunk 2]
```diff
@@ -33,16 +33,16 @@ jobs:
     - name: Updating the Release URL in POM
```
Comment (syedsalman3753) [inline .github/workflows/release-changes.yml:33]: This workflow is.wrong please update with the latest one

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: .github/workflows/push-trigger.yml (hunk 1 above)
Comment (Rakshitha650) [reply .github/workflows/push-trigger.yml:8]: @syedsalman3753 why? we need branches na

File: .github/workflows/release-changes.yml (hunk 2 above)
Comment (Rakshitha650) [reply .github/workflows/release-changes.yml]: updated

**Review by syedsalman3753 — APPROVED** [review]

**Pull request merged**